lib/storage/azure.js

```diff
@@ -5,8 +5,11 @@ var fs = require('fs');
 var zlib = require('zlib');
 var async = require('async');
 var utils = require('../utils.js');
+var defaultGzipBlacklist = require('../../defaultGzipBlacklist');
 var verbose = false;
+var _ = require('lodash');
 
+var gzip = zlib.createGzip();
```

Comment:
Whoa, you rolled back my fix to the bug where a single instance of the zlib stream was reused; what else did you roll back when you merged?